### ISSUE TITLE
Incomplete resources present a rather bleak dashboard

### ISSUE THREAD

**Issue by hubertp** [enso-org/enso#7438]
### What type of issue is this?

Permanent – Occurring repeatably

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

Trying out a nightly (7.31) reports a small number of errors about missing resources. That then leads to the dashboard not showing properly (see screenshot):
```
Resource '/esbuild' not found.
[info] [2023-07-31T13:00:38.984Z] [org.enso.projectmanager.boot.ProjectManager$] Starting Project Manager...
[info] [2023-07-31T13:00:39.207Z] [org.enso.projectmanager.boot.ProjectManager$] Started server at 127.0.0.1:30535, press enter to kill server
[info] [2023-07-31T13:00:39.599Z] [org.enso.projectmanager.protocol.ClientController] Client connected to Project Manager [1885c7be-f4a6-428d-be38-9b9e9b62c82b]
[info] [2023-07-31T13:00:40.626Z] [org.enso.projectmanager.protocol.ClientController] Client connected to Project Manager [c27e9778-f38b-4c51-b64a-a6ccc7c0a12f]
Resource '/new-project.svg' not found.
Resource '/spreadsheets.png' not found.
Resource '/geo.png' not found.
Resource '/visualize.png' not found.
Resource '/project.svg' not found.
```

### Expected behaviour

Probably all resources should be packaged.

### How we can reproduce it?

Try out latest AppImage nightly.

### Screenshots or screencasts

![Screenshot from 2023-07-31 15-01-28](https://github.com/enso-org/enso/assets/292128/4151bd7b-2238-4425-8705-a499a19d79bc)

### Enso Version

828d160c565e3ee2f25030a87fee795243e91379

### Browser or standalone distribution

Standalone distribution (local project)

### Browser Version or standalone distribution

7.31 nightly

### Operating System

Linux

**Comment by somebody1234** [comment]
oops :(
this change i made is the culprit:
https://github.com/enso-org/enso/commit/e4357f889026fbcb989398668909387f332c91be#diff-a76fe5b3adc03e5ca1e447c4b90da8455de1c0302734f4a0442f4820ff77ca44L111-R111

in particular, it adds a unique suffix to `.png` files

i *think* there are two relatively simple fixes:
- add a html loader plugin into the build process (extra dependency?)
- add a custom plugin to make these `.png` files load using the old `copy` loader (no extra dependency, preserves nice names)

**Comment by somebody1234** [comment]
@hubertp A fix is now available at #7442 - you may want to test it out

**Comment by PabloBuchu** [comment]
@hubertp are you still using the old dashboard? Is there a reason for that? Just asking to know better is there anything we could improve / fix in the new one
